eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -190,7 +252,7 @@ function Add-IssueLabels {
   }
 
   $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/labels"
-  $labelAdditions = SplitMembers -membersString $Labels
+  $labelAdditions = Split-GitHubAPIParameterMembers -membersString $Labels
   $parameters = @{
     labels = @($labelAdditions)
   }
```

Comment:
For these helpers should they just call Update-GitHubIssue?

Reply:
They don't do the same thing. This is supposed to append lables. `Update-GitHubIssue` will replace the existing labels.

Reply:
Unless I'm not understanding something then this API will replace the set of labels not append new labels. See https://developer.github.com/v3/issues/#update-an-issue

Reply:
Nevermind I was looking at the wrong docs. Your calling https://developer.github.com/v3/issues/labels/#add-labels-to-an-issue which does seem to add incrementally and not replace the entire set. I think you are correct then they are different operations.